src/arduino/components/Pressure.cpp

```diff
@@ -0,0 +1,50 @@
+#include "pressure.h"
+
+/**
+ * Returns pressure in PSI
+ *
+ * LOGIC:
+ * 1. Read sensor value
+ * 2. Convert sensor value to voltage
+ * 3. Convert voltage to pressure
+ * 4. Set pressure
+ * 5. Return pressure
+ */
+float Pressure::getPressure()
+{
+  double sensorRead = analogRead(sensorPin) * 0.0049;                                                                // Volts
+  double pressure = (((sensorRead - 0.1 * VOLTAGE_CONST) * (P_MAX - P_MIN)) / (0.8 * VOLTAGE_CONST)) + P_MIN + 0.36; // PSI
```

Comment:
Again we are pulling constants out of nowhere

Reply:
tbh its dev's code so I also have no clue

Reply:
https://forum.digikey.com/t/topic/7152
according to this japanese guy the equation used is 
(Vread - Vout_min) * (pressure_range) / (Vout_max - Vout_min)
So reverse engineering the constants this is what I get
pressure_range = 145
Vout_max = 4.5
Vout_min = 0.5
There is also the last value of 0.36 I think that is just some calibration to make sure the 0 psi is regular room pressure of 14.7psi